jax_unirep/__init__.py

```diff
@@ -1,4 +1,4 @@
 """Top-level imports for jax-unirep."""
 
-from .evotuning import evotune
+from .evotuning import evotune, fit
```

Comment:
Great stuff, this definitely should be exported at the highest level of API because it's user-facing.